templates/kubernetes/compare.html

```diff
@@ -0,0 +1,400 @@
+          <th class="u-align--center">Red Hat Openshift</th>
+          <th class="u-align--center">Rancher</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td aria-label="Category">CNCF Conformant</td>
+          <td aria-label="Canonical Kubernetes" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+          <td aria-label="Red Hat Openshift" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+          <td aria-label="Rancher" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+        </tr>
+        <tr>
+          <td aria-label="Category">High availability</td>
+          <td aria-label="Canonical Kubernetes" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+          <td aria-label="Red Hat Openshift" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+          <td aria-label="Rancher" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+        </tr>
+        <tr>
+          <td aria-label="Category">Enterprise update control</td>
+          <td aria-label="Canonical Kubernetes" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+          <td aria-label="Red Hat Openshift" class="u-align--center">
+            <span>&ndash;</span>
+          </td>
+          <td aria-label="Rancher" class="u-align--center">
+            <span>&ndash;</span>
+          </td>
+        </tr>
+        <tr>
+          <td aria-label="Category">Automated upgrades between versions</td>
+          <td aria-label="Canonical Kubernetes" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+          <td aria-label="Red Hat Openshift"class="u-align--center" >
+            <span>&ndash;</span>
+          </td>
+          <td aria-label="Rancher" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
+          </td>
+        </tr>
+        <tr>
+          <td aria-label="Category">Edge support</td>
+          <td aria-label="Canonical Kubernetes">
+            <div class="u-align--center">
+              <span class="p-rating p-rating--5"></span>
+              <p>MicroK8s</p>
+            </div>
+          </td>
+          <td aria-label="Red Hat Openshift">
+            <div class="u-align--center">
+              <span class="p-rating p-rating--3"></span>
```

Comment:
Those won't mean anything to people using screen readers. I think we should add 
`role="img"` and `aria-label`  to improve accessibility.
Like so 
`<span class="p-rating p-rating--3" role="img" aria-label="3 out of 5"></span>`

Reply:
Thanks, great spot! I'll get this fixed now. We will leave the CSS as is due to the half ratings (as discussed).

Reply:
@ClementChaumel Added those aria-labels ready for a re-review